app.js

```diff
@@ -1190,6 +1190,25 @@ define(function(require) {
 
 			monster.ui.tooltips(parent);
 
+			parent.find('#unassigned_select_all').on('click', function() {
+				var $this = $(this);
+				var $selectAllLabel = $('.unassigned_select_all_label');
```

Comment:
As a general rule, we prefer to do a lookup from an existing jQuery object when possible rather than creating a new one.
```suggestion
				var $selectAllLabel = parent.find('.unassigned_select_all_label');
```